### ISSUE TITLE
Feature Requests: Fetch volume information and Offset in DLS Logs, Adjustable fetch amount

### ISSUE THREAD

**Issue by Aethereal00** [duosecurity/duo_log_sync#28]
Hi there!
Id like to submit a request for future updates the DLS, specifically a bit more information in the logging and options in the config.
In the DLS logs it would be really nice to see the offset timestamp (it can be a pain to read the string in the offset file).
Also it would be great to get an estimation of the total  log volume still to fetch. I dont know if the API supports this, but seeing "auth producer: adding 1000 logs to the queue" gives no indication of how many batches are pending.

On a similar note I understand the API limitation of 120 seconds between requests, is there also a limit of 1000 records per request?
When catching up after an interruption in DLS forwarding it can take a very log time to retrieve all of the auth logs at 1000 logs per request, even with our offset at only 3 days.

Thank you!

**Comment by MarkTripod-Duo** [comment]
You are correct that the maximum number of log entries that can be retrieved in a single polling cycle is 1,000. This is a restriction of the Duo Admin API itself. (https://duo.com/docs/adminapi#logs)
